tests/query_errors/window_event_in_target/error.txt

```diff
@@ -0,0 +1,3 @@
+Error: 
+    9 |   "min": math::min(event.number + 1, 0),
+      |                    ^^^^^^^^^^^^ References to `event` or `$` are not allowed in this context
```

Comment:
Can we add a `NOTE` on that explains this a bit more as to why and some explanation on how to get around this? (use grouping and/or aggregate functions)

Reply:
good point